docs/release-notes/2.1.0.asciidoc

```diff
@@ -0,0 +1,54 @@
+:issue: https://github.com/elastic/cloud-on-k8s/issues/
+:pull: https://github.com/elastic/cloud-on-k8s/pull/
+
+[[release-notes-2.1.0]]
+== {n} version 2.1.0
+
+[[feature-2.1.0]]
+[float]
+=== New features
+
+* Allow predicates to be disabled on a case-by-case basis via annotation. {pull}5284[#5284] (issue: {issue}2092[#2092])
```

Comment:
```suggestion
* Allow predicates to be disabled on a case-by-case basis through annotation. {pull}5284[#5284] (issue: {issue}2092[#2092])
```
Let's try to avoid latinisms for accessibility reasons.